docs/modules/core/pages/vm-migrate-capability-matrix.md

```
@@ -0,0 +1,232 @@
+    boolean isSupportWithDataVolume();
+    boolean isSupportWithSnapshot();
+    boolean isSupportLive();
+    boolean isSupportOffline();
+    boolean isSupportSameStorage();
+    boolean needMigration(VolumeVO vo, String dstPsUuid, String dstHostUuid, List<String> systemTags);
+}
+```
+
+`StorageMigrationApiInterceptor.validate(APIPrimaryStorageMigrateVmMsg)` 在循环每个 `volumeMigrationSpecs` 项时，按 `pluginRegistry.getExtensionList(VmStorageMigrationMetric.class)` 的顺序遍历所有 metric：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)